affiliatewp-allow-own-referrals.php

```diff
@@ -5,18 +5,439 @@
+
+		// Register the activation hook.
+		register_activation_hook( $this->file, array( $this, 'install' ) );
+	}
+
+	/**
+	 * Runs the install, usually on an activation hook.
+	 *
+	 * @since 1.0.3
+	 */
+	public function install() {
+
+		// Bootstrap to include all of the necessary files
+		$this->bootstrap();
+
+		// Run other install code ...
+	}
+
+	/**
+	 * Bootstraps the actual plugin file.
+	 *
+	 * @since 1.0.3
+	 */
+	public function bootstrap() {
+		AffiliateWP_Allow_Own_Referrals::instance( $this->file );
+	}
+
+	/**
+	 * Plugin specific URL for an external requirements page.
+	 *
+	 * @since 1.0.3
+	 *
+	 * @return string URL to an externally-hosted minimum requirements document.
+	 */
+	private function unmet_requirements_url() {
+		return 'https://affiliatewp.com/add-ons/official-free/allow-own-referrals/';
+	}
+
+	/**
+	 * Outputs plugin-specific text to quickly explain what's wrong (in the plugins list table).
+	 *
+	 * @since 1.0.3
+	 *
+	 * @return string Message to explain that partial activation is in effect.
+	 */
+	private function unmet_requirements_text() {
+		esc_html_e( 'This plugin is not fully active.', 'affiliatewp-allow-own-referrals' );
+	}
+
+	/**
+	 * Retrieves plugin-specific text to describe a single unmet requirement.
+	 *
+	 * @since 1.0.3
+	 *
+	 * @return string Message for a single unmet requirement.
+	 */
+	private function unmet_requirements_description_text() {
+		return esc_html__( 'Requires %s (%s), but (%s) is installed.', 'affiliatewp-allow-own-referrals' );
+	}
+
+	/**
+	 * Retrieves plugin-specific text to describe a single missing requirement.
+	 *
+	 * @since 1.0.3
+	 *
+	 * @return string Message for a single missing requirement.
+	 */
+	private function unmet_requirements_missing_text() {
+		return esc_html__( 'Requires %s (%s), but it appears to be missing.', 'affiliatewp-allow-own-referrals' );
+	}
+
+	/**
+	 * Retrieves plugin-specific text used to link to an external requirements page.
+	 *
+	 * @since 1.0.3
+	 *
+	 * @return string Label to use when linking to the externally-hosted minimum requirements document.
+	 */
+	private function unmet_requirements_link() {
+		return esc_html__( 'Requirements', 'affiliatewp-allow-own-referrals' );
+	}
+
+	/**
+	 * Retrieves plugin-specific aria label text to describe the requirements link.
+	 *
+	 * @since 1.0.3
+	 *
+	 * @return string
+	 */
+	private function unmet_requirements_label() {
+		return esc_html__( 'AffiliateWP - Allow Own Referrals Requirements', 'affiliatewp-allow-own-referrals' );
+	}
+
+	/**
+	 * Retrieves plugin-specific text used in CSS to identify attribute IDs and classes.
+	 *
+	 * @since 1.0.3
+	 *
+	 * @return string
+	 */
+	private function unmet_requirements_name() {
+		return 'affiliatewp-allow-own-referrals-requirements';
+	}
 
-function affwp_allow_own_referrals_tracking_override( $ret, $affiliate_id ) {
+	/** Agnostic Methods ******************************************************/
```

Comment:
Use `/*` for the comment. It seems like a nitpicky thing but if we ever parse these docs into some kind of reference, it'll almost certainly trip on this thinking it's a function. 😉 
```suggestion
	/* Agnostic Methods ******************************************************/
```

Reply:
FYI this came directly from the boilerplate. It probably should be updated there, too.